### DISCUSSION TITLE
Changing vendor name to "The AdaptiveCpp project"?

### DISCUSSION THREAD

**Discussion by Arpan3323** [AdaptiveCpp/AdaptiveCpp#1580] [Q&A] [answered]
Hi all,
currently, using `plat.get_info<sycl::info::platform::vendor>()` returns "The hipSYCL project"
 
I wanted to ask if it would be appropriate to change it so it returns "The AdaptiveCpp project"?

This is the code I have:
```C++
#include<CL/sycl.hpp>
using namespace cl;

int main()
{
    std::vector<sycl::platform> platforms = sycl::platform::get_platforms();
    for (const auto &plat : platforms)
    {
        std::cout << "Platform: " << plat.get_info<sycl::info::platform::name>() << " "
        << plat.get_info<sycl::info::platform::vendor>() << " "
        << plat.get_info<sycl::info::platform::version>() << " "
        << std::endl;
    }
    
    return 0;
}
```

I believe [this](https://github.com/AdaptiveCpp/AdaptiveCpp/blob/8cf7a90232d065b80813eedfae678518fa67eed7/include/hipSYCL/sycl/platform.hpp#L155) needs to be changed if we want to return "The AdaptiveCpp project"

**Comment by illuhad** [comment, marked as answer]
Yes, good find! We should change this to "The AdaptiveCpp project". Please create a pull request :)

We currently also still identify the OpenMP host device as "hipSYCL OpenMP host device", and the device vendor as "hipSYCL project" in the two function definitions here:
https://github.com/AdaptiveCpp/AdaptiveCpp/blob/8cf7a90232d065b80813eedfae678518fa67eed7/src/runtime/omp/omp_hardware_manager.cpp#L40

If you create a PR, you might also want to change this accordingly :)

**Reply by Arpan3323** [reply]
I will take care of those too, thanks for letting me know.
